internal/cli/environment/create/local_registry.go

```diff
@@ -0,0 +1,81 @@
+package create
+
+import (
+	"context"
+	"io"
+	"os"
+
+	"capact.io/capact/internal/cli/printer"
+	"github.com/docker/docker/api/types"
+
+	"capact.io/capact/internal/cli/config"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+)
+
+const (
+	// ContainerRegistryName defines name for the container registry. It's also used as DNS for registry.
+	ContainerRegistryName = "capact-registry.localhost"
+	// ContainerRegistryPort defines port for the container registry.
+	ContainerRegistryPort = "5000"
```

Comment:
It could be useful to have this configurable as we bind this port on the host, although I can be skipped for now IMO.

Reply:
I tried to expose opt to set different port and the code is available here: https://github.com/capactio/capact/pull/475/commits/489dfbb9a3dd4c0eb1e580d272dec2f25aab6334

but I was not able to make it working with k3d cluster, as we run out of time, I will extract that point to a follow-up 👍